test/memspaces/memspace_numa.cpp

```diff
@@ -235,3 +235,149 @@ TEST_F(memspaceNumaProviderTest, allocFree) {
     ret = umfMemoryProviderFree(hProvider, ptr, size);
     ASSERT_EQ(ret, UMF_RESULT_SUCCESS);
 }
+
+TEST_F(numaNodesCapacityTest, CapacityFilter) {
+    if (capacities.size() <= 1) {
+        GTEST_SKIP() << "Not enough numa nodes";
```

Comment:
`"Not enough numa nodes - skipping the test ... "`

Reply:
"Not enough numa nodes - skipping the test"

Reply:
done